components/audit/src/main/java/org/trellisldp/audit/DefaultAuditService.java

```diff
@@ -41,6 +43,7 @@
  * 
  * @author acoburn
  */
+@ApplicationScoped
```

Comment:
Can/Should @ApplicationScoped be on the NoopAuditService also? And possibly all the Noop*Service's.